test/src/e2e_vm_tests/test_programs/should_pass/dca/impl_unused_fn/test.toml

```diff
@@ -0,0 +1,18 @@
+category = "compile"
+
+# check: $()struct Data1 {
+# nextln: $()This struct is never used.
+
+# check: $()fn get(self) -> u64 {
+# nextln: $()
+# nextln: $()
+# nextln: $()
+# nextln: $()This method is never called.
+
+# check: $()fn get2(self) -> u64 {
+# nextln: $()
+# nextln: $()
+# nextln: $()
+# nextln: $()This method is never called.
```

Comment:
I tried this example in Rust and I got the following
<img width="494" alt="image" src="https://user-images.githubusercontent.com/59666792/227233355-463974db-ea3c-4540-abc6-2244843045dd.png">
which is close enough to what we now have! Very nice.